qubes_config/global_config.glade

```diff
@@ -5813,7 +5718,7 @@ Files cannot be copied to dom0.</property>
                                 <property name="visible">True</property>
                                 <property name="can-focus">True</property>
                                 <property name="halign">start</property>
-                                <property name="label" translatable="yes">To learn more about Open in VM policy and its uses, you can check the &lt;a href="https://github.com/Qubes-Community/Contents/blob/master/docs/common-tasks/opening-urls-in-vms.md"&gt;online documentation&lt;/a&gt;. 
+                                <property name="label" translatable="yes">To learn more about Open in VM policy and its uses, you can check the &lt;a href="https://github.com/Qubes-Community/Contents/blob/master/docs/common-tasks/opening-urls-in-vms.md"&gt;online documentation&lt;/a&gt;.
```

Comment:
Will change depending on the answer to https://github.com/QubesOS/qubes-desktop-linux-manager/pull/150#issuecomment-1529486150.